data/beaconrestapi/src/test/java/tech/pegasys/artemis/beaconrestapi/handlers/validator/PostDutiesTest.java

```diff
@@ -58,19 +58,20 @@ public void shouldReturnBadRequestWhenNegativeEpochNumberInBody() throws Excepti
   }
 
   @Test
-  public void shouldReturnEmptyListWhenNoValidatorDuties() throws Exception {
```

Comment:
have we removed this test and re-tasked it?
the intent of this test was to show that an empty list is returned on an empty list passed in, and that's to be consistent with another API, so we should retain that intent in the testing.

Reply:
It looked to me as if we were using the empty list to just represent no data, so I cut this since we're now using `Optional`'s to represent no data.  I've gone back and added some short-circuit logic that will return an empty list if an empty list of public keys are provided, and added some more tests.